package.json

```diff
@@ -1,25 +1,25 @@
 {
   "name": "vscode-commons",
   "displayName": "Red Hat common extension",
-  "description": "",
+  "description": "Telemetry Middleware",
```

Comment:
Base utilities for Red Hat extensions. Notably controls Telemetry settings.

Reply:
Updated